Include/cpython/pystate.h

```diff
@@ -134,7 +134,8 @@ struct _ts {
 
     /* Unique thread state id. */
     uint64_t id;
-
+    size_t stack_space;
```

Comment:
I would prefer to only declare these 2 attributes on macOS (`#ifdef __APPLE__`).